Assets/Scripts/Levels/LevelManager.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class LevelManager : MonoBehaviour
+{
+    [SerializeField]
+    private Button[] levelBtns;
+    [SerializeField]
+    private GameObject _loader;
+    private Animator anim;
+    // Start is called before the first frame update
+    void Start()
+    {
+        anim = GetComponent<Animator>();
+        int levelAt = PlayerPrefs.GetInt("levelAt", 2);
+        for (int i = 0; i < levelBtns.Length; i++)
```

Comment:
I think this is not a generic code please check this if it works for only four levels and we modify 100 levels then check if this code works.